src/qutipinterface.jl

```diff
@@ -0,0 +1,119 @@
+@doc """
+Interface for `sesolve`[http://qutip.org/docs/3.1.0/apidoc/functions.html#module-qutip.sesolve] of QuTiP in QuDynamics
+
+`sesolve` in QuDynamics, extended version.
+
+### Arguments
+
+Inputs :
+* H/SE/LVNE :: AbstractQuArray/QuSchrodingerEq/QuLiouvillevonNeumannEq Types
+
+  This parameter supports Hamiltonian of the system, Schrodinger Equation, LiouvillevonNeumann Equation
+* psi :: AbstractQuArray
+
+  Initial state vector or density matrix of the system
+* tlist
+
+  Time step array/range
+* solver
+
+  Keyword for selecting the solver from the various solvers supported by QuDynamics
+
+Output :
+* Array of tuples, which specify the time-step and the corresponding evolved state
+""" ->
+function sesolve(H::QuBase.AbstractQuArray, psi::QuBase.AbstractQuArray, tlist, solver=QuExpmV())
+    evolved_state = Array(Tuple, length(tlist)-1)
```

Comment:
Using an array of tuples is probably not very economic. I think having separate Arrays for times and states is better (and more consistent).